### ISSUE TITLE
vaadinCompileWidgetset with gwt.jvmArgs produces "No main class found" but sucessfull build. 

### ISSUE THREAD

**Issue by deimel** [johndevs/gradle-vaadin-plugin#163]
Setting gwt.jvmArgs leads to a wrong string concatenation in class CompileWidgetsetTask.groovy
which produces a "no main class found" in java.lang.String.

Unfortunately the var. "widgetsetCompileProcess"  is not logged out and the build is successfull

gwt {
      // [ERROR] Fehler: Hauptklasse [Ljava.lang.String;@13fec43 konnte nicht gefunden ...werden
      jvmArgs = ' -Xms500M '
      logLevel = 'DEBUG'
      logging = true
}

Doing the concatenation like it is done with gwt.extraArgs should solve the problem

**Comment by deimel** [comment]
An "as List" should solve the problem
```
def widgetsetCompileProcess = ['java']
if (project.vaadin.gwt.jvmArgs) {
   widgetsetCompileProcess += project.vaadin.gwt.jvmArgs as List
}
```

**Comment by deimel** [comment]
Addendum:

Using a String array like
```
jvmArgs = [ ' -Xms500M ']
```
(how it would be correct) doesn't solve the problem

**Comment by johndevs** [comment]
Fixed in a8c52dfde665c4c90707ed3cb725c006ec6b3ab9
